src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+function average(ages, persons) {
```

Comment:
What is `average`?
Use more descriptive names. 
If you do some action in function, then describe it. Use verb as first word in name.
`calculateAverageAges` or `getAverageAges` would be better